src/System.Reflection/tests/Assembly/Assembly_GetEntryAssembly.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using System;
+using System.Reflection;
+
+namespace System.Reflection.Tests
+{
+    public class AssemblyGetEntryAssemblyTest
+    {
+        [Fact]
+        public void Test_GetEntryAssembly()
+        {
+            Assert.NotNull(Assembly.GetEntryAssembly());
+            Assert.True(Assembly.GetEntryAssembly().ToString().StartsWith("xunit.console.netcore", StringComparison.OrdinalIgnoreCase));
```

Comment:
I wonder If there's a way we could use an entry assembly that we know won't ever change? I wouldn't be surprised if someday we switch to the "regular" xUnit runner, which would just have a name of "xunit.console". Is it easy to re-use some of the helpers from the Process tests to start a program with a well-known name? The way you have it here works fine, but it would break if we switch test runners down the line.